### PR TITLE
Drivetrain motor change

### PR DESCRIPTION
**Pull request by Lemi27**
-changed motors in Drivetrain.java from Spark Max to Talon

### CODE REVIEW

File: src/main/java/frc/robot/commands/drivetrain/Drive [hunk 1]
```diff
@@ -0,0 +1,65 @@
+/* ==================================================
```
Comment (danielzyy) [inline src/main/java/frc/robot/commands/drivetrain/Drive]: I don't think you need to keep including this file, you've already updated Drive.java, so you can just delete this file. You also don't have to create a new PR every time you remove a file, just delete it and push it to the same branch and it should update the PR.

Comment (bricktrooper) [reply src/main/java/frc/robot/commands/drivetrain/Drive]: I think this file is supposed to be `Drive.java` no?  Just missing the extension.

**Comment by bricktrooper** [comment]
You need to do a merge from `master` into this branch to get your changes from this PR: https://github.com/team4015/FRC-Robot-2022-Rapid-React/pull/25

Right now this PR doesn't have your updated steering changes so you might have merge conflicts.

File: src/main/java/frc/robot/commands/drivetrain/Drive (hunk 1 above)
Comment (danielzyy) [reply src/main/java/frc/robot/commands/drivetrain/Drive]: @bricktrooper it looks like an older version of Drive.java which has already been updated from this version to a newer version that includes the turn speed in the `setMotors()` function. So it is ok to discard it.

**Review by Geotricity — APPROVED** [review]
Looks good now now that Drive is removed.

**Pull request merged**